### ISSUE TITLE
Issue when i call to bitcoin_testnet rpc

### ISSUE THREAD

**Issue by Sotatek-MinhNguyen11** [btcsuite/btcd#2160]
![Screenshot from 2024-04-08 11-27-22](https://github.com/btcsuite/btcd/assets/146687333/e8f1b0cc-5ff5-4c3a-80a6-c778229e5c7c)
Make sure that the user and password are correct

**Comment by guggero** [comment]
Wrong repo? `btcd` is written in Go, your code seems to be Rust. Or are you trying to connect to `btcd` from Rust?
More details please.

**Comment by Sotatek-MinhNguyen11** [comment]
@guggero  yes, i'm trying to connect to btcd from Rust, so it's possible?

**Comment by guggero** [comment]
Yes, it should be possible. The error seems to be "Connection reset by peer". So you're using the wrong port or `btcd` isn't ready.

**Comment by Sotatek-MinhNguyen11** [comment]
@guggero hmm, i don't think so, btcd ready and I have tried all possible ports
![Screenshot from 2024-04-08 15-47-35](https://github.com/btcsuite/btcd/assets/146687333/3e607125-c4c5-43da-9e63-6954d6898645)
![Screenshot from 2024-04-08 15-51-42](https://github.com/btcsuite/btcd/assets/146687333/69a46fb6-df3b-49ec-8067-dfcb8a11e02a)

**Comment by guggero** [comment]
Then it's probably an issue with the IP address or firewall that you can't reach `btcd` from the Rust application.
